packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationsEditor.test.tsx

```diff
@@ -255,11 +255,10 @@ describe('SchemaDrivenAnnotationEditor tests', () => {
     server.use(noAnnotationsHandler, noSchemaHandler)
 
     await renderComponent()
-    await screen.findByText('has no annotations', { exact: false })
     const buttons = await screen.findAllByRole('button')
 
-    // Expect one button to add annotations, and another to save
-    expect(buttons.length).toBe(2)
+    // Renders a default form state with one empty key-value field when no annotations or schema are present
+    expect(buttons.length).toBe(7)
```

Comment:
Can we improve this to check for each ui element we expect?

e.g. 
Find a field with label `'Key'`, make sure that it has value `'newKey'`
Find a 'Type' field, make sure it has value 'String',
Find a 'Value(s)' field, make sure it has a blank value

Find the button to add annotations
Find the save button